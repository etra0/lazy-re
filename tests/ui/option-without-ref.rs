use lazy_re::lazy_re;

#[lazy_re]
#[repr(C, packed)]
struct Foo {
  a: usize,
  b: Option<f32>,

  #[lazy_re(offset = 0xBB)]
  c: usize
}

fn main() {}