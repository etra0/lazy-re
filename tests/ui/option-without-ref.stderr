error: We can only handle Option<&T> for now
 --> tests/ui/option-without-ref.rs:7:3
  |
7 |   b: Option<f32>,
  |   ^
